resources/less/hakija.less

```diff
@@ -446,6 +446,7 @@ textarea.application__form-text-input {
 }
 .application__form-text-area {
   width: 610px;
+  resize: none;
```

Comment:
Do we want to do this? Seems like a usability downgrade (for some)...

Reply:
By Antti's request.